File: app/src/main/java/vng/zalo/tdtai/zalo/zalo/models/Room.java
package vng.zalo.tdtai.zalo.zalo.models;

import com.google.firebase.Timestamp;

import java.util.Map;

public class Room {
    public String id;
    public String avatar;
    public Timestamp createdTime;
    public String name;

    public Map<String, RoomMember> memberMap;
}

File: app/src/main/java/vng/zalo/tdtai/zalo/zalo/models/Room.kt
package vng.zalo.tdtai.zalo.zalo.models

import com.google.firebase.Timestamp

data class Room(var id: String? = null,
                var avatarUrl: String? = null,
                var createdTime: Timestamp? = null,
                var name: String? = null,
                var memberMap: Map<String, RoomMember>? = null) {
    fun toMap(): Map<String, Any> {
        return HashMap<String, Any>().apply {
            put("createdTime", createdTime!!)
        }
    }
}